cmake/external/mkldnn.cmake

```diff
@@ -1,41 +1,28 @@
 include (ExternalProject)
 
 set(MKLDNN_URL https://github.com/intel/mkl-dnn.git)
-# If MKLDNN_TAG is updated, check if MKLML_VERSION and platform.cmake.patch need to be updated.
-set(MKLDNN_TAG v0.18.1)
-set(MKLML_VERSION 2019.0.5.20190502)
+set(MKLDNN_TAG v1.0.2)
```

Comment:
please don't remove the MKLML portions.
while mklml isn't used by mkldnn explicitly, there is a build option for using mklml by itself. 
and we produce a ort mkl nuget package for customers who want to use mklml only.

Reply:
added MKLML back to cmake.